messages/list.js

```diff
@@ -1,43 +1,38 @@
 module.exports = {
-	"commandDescription": "lists basic information about all rules matching provided criteria",
-	"commandDescriptionLong": `Lists all the rules available in the catalog. To look at a smaller set of rules,
-	use the filter options available. To get more information about a specific rule,
-	you can use the scanner:rule:describe command.`,
+	"commandDescription": "list basic information about all rules matching provided criteria",
+	"commandDescriptionLong": `Lists all the rules available in the catalog. You can filter the output to view a smaller set of rules. To get more information about a specific rule, use the \`scanner:rule:describe\` command.`,
 	"flags": {
-		"languageDescription": "language(s) to filter list by",
-		"languageDescriptionLong": "Filters the list based on the specified languages. Specify multiple languages as a comma-separated list. See the PMD CLI documentation for a list of supported languages.",
-		"categoryDescription": "categories to filter list by",
-		"categoryDescriptionLong": "Select rules by category. Enter multiple values as a comma-separated list. E.g., 'Best Practices',Performance ",
-		"rulesetDescription": "[deprecated] ruleset(s) to filter list by",
-		"rulesetDescriptionLong": "[Deprecated] Select rules by ruleset. Enter multiple values as a comma-separated list.",
-		'engineDescription': "engine(s) to filter list by",
-		'engineDescriptionLong': "Filters the list based on the specified engines. Specify multiple engines as a comma-separated list."
+		"languageDescription": "select rules by language",
+		"languageDescriptionLong": "Selects rules by language. Enter multiple values as a comma-separated list.",
+		"categoryDescription": "select rules by category",
+		"categoryDescriptionLong": "Selects rules by category. Enter multiple values as a comma-separated list.",
+		"rulesetDescription": "[deprecated] select rules by ruleset",
+		"rulesetDescriptionLong": "[deprecated] Selects rules by ruleset. Enter multiple values as a comma-separated list.",
+		'engineDescription': "select rules by engine",
+		'engineDescriptionLong': "Selects rules by engine. Specify multiple engines as a comma-separated list."
```

Comment:
@jfeingold35 Should read: "Selects rules by engine. Enter multiple values as a comma-separated list."